coordinator/src/main/scala/com/socrata/datacoordinator/secondary/SecondaryWatcher.scala

```diff
@@ -47,17 +53,26 @@ class SecondaryWatcher[CT, CV](universe: => Managed[SecondaryWatcher.UniverseTyp
         "endingDataVersion" -> job.endingDataVersion
       ) {
         log.info(">> Syncing {} into {}", job.datasetId, secondary.storeId)
+        if (job.retryNum > 0) log.info("Retry #{} of {}", job.retryNum, maxRetries)
         try {
           playbackToSecondary(secondary, job)
           log.info("<< Sync done for {} into {}", job.datasetId, secondary.storeId)
         } catch {
           case e: Exception =>
-            log.error("Unexpected exception while updating dataset {} in secondary {}; marking it as broken",
-                      job.datasetId.asInstanceOf[AnyRef], secondary.storeId, e)
-            secondaryManifest.markSecondaryDatasetBroken(job)
+            if (job.retryNum < maxRetries) {
```

Comment:
Well keep in mind most replications take many many minutes, often hours, sometimes more than a day.   The way the timing works is that after the first failure, next_retry is set to some time in the future.  The watcher loops back, it might not even take that same item up, some other item might have moved up the stack.  Thus the back off actually won't do that much.....   unless the failures occur right away.

We could certainly increase the defaults ... perhaps to 5 minutes * 5 retries = max back off of 160 minutes